attendance/attendance.py

```diff
@@ -85,13 +86,18 @@ def mark_attendance(wifi_ssid_list, credentials):
     if not check_internet_connection():
         sys.exit()
 
-    # get list of wifi ssid's
-    wifi_ssid_list = get_wifi_list()
+    # get wifi's ssid if the platform is windows
+    if _platform == "win32":
```

Comment:
I was thinking how about if we add over get_wifi_list():(https://github.com/amfoss/attendance-tracker/blob/3f7ef6cea0187d69b9a35ae9d2b968b00e411acc/attendance/attendance.py#L41 
instead of modifying here so we can add another condition like 
```
if _platform == "win32":
      .
      .
```
so it will be looking uniform